keyboards/dcpedit/midevil/rules.mk

```
@@ -0,0 +1,3 @@
+QUANTUM_PAINTER_DRIVERS += ili9488_spi
+SRC += image/background.qgf.c \
+			 image/animated.qgf.c
```

Please replace the tabs with spaces.

```suggestion
SRC += image/background.qgf.c \
       image/animated.qgf.c
```